models/demos/falcon40b/demo/demo.py

```diff
@@ -195,8 +195,8 @@ def run_falcon_demo_kv(
     prefill_on_host,
 ):
     torch.manual_seed(0)
-    for device in devices:
-        device.enable_program_cache()
+    # for device in devices:
+    #     device.enable_program_cache()
```

Comment:
Do we have to turn off program cache? Can we enable and leave what is working on `main`?